src/environments/environment.ts

```diff
@@ -14,8 +14,8 @@ export const environment = {
         appId: '1:216500507265:web:ef43d619b4ac117655f380',
         measurementId: 'G-3FKLY60L44'
     },
-    // api: 'http://localhost:8080/api',
-    api: 'https://inventurbo-api.herokuapp.com/api',
+    api: 'http://localhost:8080/api',
+    // api: 'https://inventurbo-api.herokuapp.com/api',
```

Comment:
Necessary change? I didn't see any updates to the backend in this PR.

Reply:
Forgot to mention that I push a small change to the backend. Besides that, this change allows us to modify the backend and use it locally during the iteration. I will change it back in the end before landing everything to master.